tests/json_fmt.cpp

```diff
@@ -12,62 +12,148 @@ int main_readme() {
     const std::string input =  "+.1"; // not valid
     double result;
     fast_float::parse_options options{ fast_float::chars_format::json };
-    auto answer = fast_float::from_chars_advanced(input.data(), input.data()+input.size(), result, options);
-    if(answer.ec == std::errc()) { std::cerr << "should have failed\n"; return EXIT_FAILURE; }
+  auto answer = fast_float::from_chars_advanced(
+      input.data(), input.data() + input.size(), result, options);
+  if (answer.ec == std::errc()) {
+    std::cerr << "should have failed\n";
+    return EXIT_FAILURE;
+  }
     return EXIT_SUCCESS;
 }
 
-
 int main_readme2() {
     const std::string input =  "inf"; // not valid in JSON
     double result;
     fast_float::parse_options options{ fast_float::chars_format::json };
-    auto answer = fast_float::from_chars_advanced(input.data(), input.data()+input.size(), result, options);
-    if(answer.ec == std::errc()) { std::cerr << "should have failed\n"; return EXIT_FAILURE; }
+  auto answer = fast_float::from_chars_advanced(
+      input.data(), input.data() + input.size(), result, options);
+  if (answer.ec == std::errc()) {
+    std::cerr << "should have failed\n";
+    return EXIT_FAILURE;
+  }
     return EXIT_SUCCESS;
```

Comment:
You have indentation problems here.

@lemire what do you think about adding a clang-format config file so we can avoid this type of review cycles?

Reply:
@deadalnix Please review https://github.com/fastfloat/fast_float/pull/257

I don't think we want to consider formatting mistakes as blocking for a PR. We can reformat later.